### PR TITLE
Made modifications to the dataloads components and dataload service

### PR DESCRIPTION
**Pull request by VarunReddy1111**

### CODE REVIEW

**Review by oblodgett — CHANGES_REQUESTED** [review]
node_modules should not be included in the PR

**Review by oblodgett — CHANGES_REQUESTED** [review]
package-lock.json and package.json need to get removed also

**Comment by adamgibs** [comment]
@oblodgett The package.json and package-lock.json are needed to build the node_modules.

**Comment by oblodgett** [comment]
@adamgibs not in the root of the project

**Comment by adamgibs** [comment]
@oblodgett Ah, didn't catch that. Yeah, they shouldn't be there.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "file-saver": "^2.0.5"
```
Comment (oblodgett) [inline package.json]: @adamgibs @VarunReddy1111 this file needs to be removed.

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (oblodgett) [inline package-lock.json]: @adamgibs @VarunReddy1111 this file needs to be removed.

**Review by adamgibs — APPROVED** [review]

**Review by oblodgett — APPROVED** [review]

**Pull request merged**